src/Makefile.am

```diff
@@ -356,7 +356,7 @@ endif
 
 # wallet: shared between bitcoind and bitcoin-qt, but only linked
 # when wallet enabled
-libbitcoin_wallet_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+libbitcoin_wallet_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) $(SQLITE_CFLAGS)
```

Comment:
5c19bd0f4b893a91c02e652ac8f45ef0ccdeca1c

Nice!
During reviewing #19077 I've noticed that the `SQLITE_CFLAGS` is empty, and I was not worried about adding it to the `libbitcoin_wallet_a_CPPFLAGS`. But consistency is good, it is not guaranteed the `SQLITE_CFLAGS` will be empty always :)